### DISCUSSION TITLE
Is there a way to include the cc functionality to the sent email.

### DISCUSSION THREAD

**Discussion by Ahmed-Badawy** [spatie/laravel-backup#1068] [Ideas]
I have seen the issue for sending emails to multiple addresses not implemented. 
at least can you add the cc functionality in there so we could send the notifications to multiple addresses. 

thanks for the great work.

**Comment by olssonm** [comment]
As laravel-backup uses Laravels native MailChannel you can actually supply an array for `backup.notifications.mail.to` in backup.php, like so:

```
'mail' => [
  'to' => ['recipient1@test.com', 'recipient2@test.com'],
],
```
Both adresses will be in the `To`-header however.

I haven't found a way to add CC-adresses nor use custom adresses per notification out of the box; you can however of course write your own notifiable and/or notifications ([docs](https://docs.spatie.be/laravel-backup/v6/sending-notifications/customizing-the-notifiable/)).
